hangman.js

```diff
@@ -0,0 +1,117 @@
+  // let usedLetters = [];
+
+  for (var i = 0; i < word.length; i++) {
+    underScores[i] = "_";
+  }
+  //assign innerText at answerText
+  let answerText = document.getElementById("dashes");
+  answerText.innerText = underScores.join(" ");
+
+  // form will prevnt loop from occuring when entering a letter.
+  let form = document.querySelector("form");
+  form.addEventListener("submit", event => {
+    // Show the player their progress]
+    answerText.innerText = underScores.join(" ");
+    // Get a guess from the player
+    let input = document.getElementById("input");
+    //grabbing the value entered into the input
+    let guess = input.value;
+    //clearing the input box
+    input.value = "";
+
+    event.preventDefault();
+    // function checkDupes(guess) {
+    //   if (!usedLetters.includes(userInput)) {
+    //     return true;
+    //   } else {
+    //     window.alert("This Letter Was Used Already!");
+    //   }
+    // }
+
+    //pushes guessed letters to letters guessed
+    progress.innerText += " " + guess + ",";
+    //Update the game state with the guess
+    for (let j = 0; j < word.length; j++) {
+      if (!guess.match(strCheck)) {
+        answerText.innerText = "Please Enter a Letter!";
+        break;
+      }
+      //check letters against word
+      if (word[j] === guess) {
+        underScores[j] = guess;
+        answerText.innerText = underScores.join(" ");
+        win--;
+        //checking dupes
+        // usedLetters.push(guess);
+      } //check to see if the letter is in the word
+      else if (word.includes(guess) === false) {
+        tries--;
+        livesRemaining.innerText = "You have " + tries + " left";
+        break;
+      }
+    }
+
+    //add the pictures depending on the lives left
+    let newElement = document.createElement("img");
+    switch (tries) {
+      case 4:
+        newElement.setAttribute("src", "FullSizeRender.png");
+        document.getElementById("image").appendChild(newElement);
+        break;
+      case 3:
+        // let newElement = document.createElement("img");
+        newElement.setAttribute("src", "FullSizeRender-1.png");
+        document.getElementById("image").appendChild(newElement);
+        break;
+      case 2:
+        // let newElement = document.createElement("img");
+        newElement.setAttribute("src", "FullSizeRender-2.png");
+        document.getElementById("image").appendChild(newElement);
+        break;
+      case 1:
+        // let newElement = document.createElement("img");
+        newElement.setAttribute("src", "FullSizeRender-3.png");
+        document.getElementById("image").appendChild(newElement);
+        break;
+      case 0:
+        // let newElement = document.createElement("img");
+        newElement.setAttribute("src", "FullSizeRender-4.png");
+        document.getElementById("image").appendChild(newElement);
+        break;
+    }
+    //checking if the word has no remaining letters
+    if (win === 0) {
```

Comment:
This logic causes glitch where user put same correct letter multiple times and win the game without guessing other letters correctly.